### ISSUE TITLE
EMR: Support multiple master nodes

### ISSUE THREAD

**Issue by mattburgess** [terraform-providers/terraform-provider-aws#9226]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Since EMR-5.23.0 it's possible to specify that you want multiple master nodes in an EMR cluster for HA. 

I couldn't immediately see a way of requesting that via Terraform.

### New or Affected Resource(s)

* aws_emr_cluster
  * master_instance_group

### Potential Terraform Configuration

```hcl
resource "aws_emr_cluster" "example" {
    master_instance_group {
        instance_type = "m4.large"
        instance_count = "3"
    }
}
```

### References

* https://aws.amazon.com/about-aws/whats-new/2019/05/amazon-emr-now-supports-multiple-master-nodes-to-enable-high-availability-for-hbase-clusters/
* https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-ha-launch.html

**Comment by bflad** [comment]
Indeed! I have submitted an update to the `aws_emr_cluster` resource here to support the new `instance_count` argument for the `master_instance_group` configuration block: #9235

**Comment by bflad** [comment]
Support for this has been merged and will release with version 2.20.0 of the Terraform AWS Provider, later this week. 👍
